code/WorkInProgress/laundry.dm

```diff
@@ -210,32 +207,12 @@
 	else //Prevents stuffing more than one person in at a time
 		user.visible_message("<span class='alert'>[user] tries to shove [W.affecting] into the laundry machine but it was already running.</span>")
 
-/obj/submachine/laundry_machine/proc/generate_html()
-	src.HTML = "<center><big><b>WashMan 550</b></big></center><hr><br>"
-	if (src.on && src.cycle != POST)
-		src.HTML += "<b>STATUS: [src.cycle == DRY ? "Drying" : "Washing"]</b><br>Please wait, machine is currently running."
-		return
-	else
-		if (src.cycle == POST)
-			src.HTML += "<b>STATUS: Cycle Complete</b><br>"
-		else
-			src.HTML += "<b>STATUS: Idle</b><br>"
-		src.HTML += "CYCLE: <a href='byond://?src=\ref[src];cycle=1'>[src.on ? "Stop" : "Start"]</a><br>"
-		src.HTML += "DOOR: <a href='byond://?src=\ref[src];door=1'>[src.open ? "Close" : "Open"]</a><br>"
-
-/obj/submachine/laundry_machine/proc/show_window(mob/user)
-	if (!user)
-		return
-	if (!src.HTML)
-		src.generate_html()
-	user.Browse(src.HTML, "window=laundry_machine;size=300x200;title=[capitalize(src.name)]")
-
 /obj/submachine/laundry_machine/mouse_drop(over_object,src_location,over_location)
 	var/mob/user = usr
 	if (!user || !over_object || BOUNDS_DIST(user, src) > 0 || BOUNDS_DIST(user, over_object) > 0 || is_incapacitated(user) || (issilicon(user) && BOUNDS_DIST(src, user) > 0))
 		return
-	if (src.on)
-		src.visible_message("[user] tries to open [src]'s door, but [src] is running and the door is locked!")
+	if (src.on || !src.open)
+		src.visible_message("[user] tries to open [src]'s door, but the door is closed!")
```

Comment:
that message doesn't make sense, of course the door is closed if you're trying to open it!

Reply:
TGUI is bad for my health